db/seeds.rb

```diff
@@ -45,7 +44,7 @@
               details: Faker::Lorem.sentence(word_count: 50, supplemental: true, random_words_to_add: 50),
               prefecture: ["Tokyo", "Kanagawa", "Chiba", "Saitama", "Ibaraki", "Tochigi", "Yamanashi"].sample,
               price: ((rand(5..20)) * 1000),
-              user_id: [user1, user2, user3, user4].sample.id
+              user_id: [user1, user3, user5, user6].sample.id
```

Comment:
I might be completely wrong but here maybe you need user1, user3, user5 and user7 - not user6. user7 is Shante as a provider whereas user6 is Carl as a renter, and for Kondo creation it should probably sample among the providers only?

Reply:
Thanks for catching that Cédrine. I also noticed that the comments I made were wrong. Will make the changes.